doc/Building Tribler on Windows.md

```diff
@@ -9,7 +9,7 @@ To build a Tribler installer, you'll need some additional scripts and packages.
 * The git command tools (version 2.7.0) are required to fetch the latest release information. These can be downloaded from [here](https://git-scm.com/download/win). 
 * Py2Exe (0.6.9), a tool to create an executeable from python files. Grab the latest version [here](http://sourceforge.net/projects/py2exe/files/py2exe/).
 * The builder needs to find all packages that are required by Tribler so make sure you can run Tribler on your machine and that there are no missing dependencies.
-* Nullsoft Scriptable Install System (NSIS) (version 2.5.0) is a script-driven Installer authoring tool for Microsoft Windows with minimal overhead. It can be downloaded [here](http://nsis.sourceforge.net/Download).
+* Nullsoft Scriptable Install System (NSIS) (version 2.5.0) is a script-driven Installer authoring tool for Microsoft Windows with minimal overhead. It can be downloaded [here](http://nsis.sourceforge.net/Download). As we have experienced problems with the current beta of NSIS 3, we recommend using a stable version. As of writing this is version 2.5.0.
```

Comment:
I meant explaining the specific error we had (as in the uninstall functions weren't been called) so the next guy can test if it works or not with a more recent release.